src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/LocalDBAPI.cs

```diff
@@ -300,7 +300,7 @@ internal static void CreateLocalDBInstance(string instance)
                     if (s_configurableInstances == null)
                     {
                         Dictionary<string, InstanceInfo> tempConfigurableInstances = new Dictionary<string, InstanceInfo>(StringComparer.OrdinalIgnoreCase);
-                        object section = PrivilegedConfigurationManager.GetSection("system.data.localdb");
+                        object section = ConfigurationManager.GetSection("system.data.localdb");
```

Comment:
I'm just not sure why a privileged config manager was needed in .NET Framework, and if removing it has some consequences. Could you provide more details here?

Reply:
PrivilegedConfigurationManager.GetSection was simply calling ConfigurationManager.GetSection. I just cut out the middle man. I couldn't see anything in PrivilegedConfigurationManager that would make is behave differently.